src/main/java/com/icthh/xm/tmf/ms/customer/web/rest/errors/BadRequestAlertException.java

```diff
@@ -1,13 +1,13 @@
 package com.icthh.xm.tmf.ms.customer.web.rest.errors;
 
-import org.zalando.problem.AbstractThrowableProblem;
-import org.zalando.problem.Status;
+import com.icthh.xm.commons.exceptions.BusinessException;
 
 import java.net.URI;
 import java.util.HashMap;
 import java.util.Map;
 
-public class BadRequestAlertException extends AbstractThrowableProblem {
+// TODO replace by BusinessException
```

Comment:
is it possible to solve this TODO before merge?

Reply:
Honestly not clear for me this comment. Doest it mean remove BadRequestAlertException at all and replace all usages by BusinessException?

Reply:
Returned back previous realization. Not sure why this was changed